learners/setup.md

```diff
@@ -2,52 +2,101 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+## Setup
 
-## Data Sets
+You'll need a few things set up before starting the course:
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+* the course materials should be downloaded to your computer
+* the Python libraries used in the course should be installed
 
-## Software Setup
+We'll assume you have some familiarity with the command line:
 
-::::::::::::::::::::::::::::::::::::::: discussion
+* you can run commands
+* you can navigate directories with `cd`
+* you can inspect directories with `ls` (or `dir` if you're using Command Prompt on Windows)
 
-### Details
 
-Setup for different systems can be presented in dropdown menus via a `spoiler`
-tag. They will join to this discussion block, so you can give a general overview
-of the software used in this lesson here and fill out the individual operating
-systems (and potentially add more, e.g. online setup) in the solutions blocks.
+### Getting the course materials
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::
+The course materials are hosted on GitHub and you'll need `git` installed to access them.
+Fortunately `git` is free!
 
-:::::::::::::::: spoiler
+Try opening a terminal window and running:
 
-### Windows
+```bash
+> git --version
```

Comment:
Do we expect most students to be on a linux-compatible OS, or on Windows?

If linux-compatible, I'd recommend choosing a placeholder prompt that doesn't have the special meaning in linux shells (and risk of entering it accidentally) that ">" does. Consider percent "%" or colon ":" as an alternative.